File: VB/AnimationExample/MainForm.vb
Imports DevExpress.XtraCharts
Imports System
Imports System.Windows.Forms

Namespace AnimationExample
    Partial Public Class MainForm
        Inherits DevExpress.XtraEditors.XtraForm

        Private ReadOnly Property series() As Series
            Get
                Return chartControl.Series(0)
            End Get
        End Property
        Private ReadOnly Property view() As PieSeriesView
            Get
                Return TryCast(series.View, PieSeriesView)
            End Get
        End Property

        Public Sub New()
            InitializeComponent()
        End Sub

        #Region "#AnimationConfig"
        Protected Overrides Sub OnLoad(ByVal e As EventArgs)
            MyBase.OnLoad(e)

            view.Animation = New GripPieAnimation With { _
                .BeginTime = New TimeSpan(0, 0, 0, 0), .PointDelay = New TimeSpan(0, 0, 0, 0, 250), .Duration = New TimeSpan(0, 0, 0, 1, 500), .EasingFunction = New LinearEasingFunction With {.EasingMode = EasingMode.Out} _
            }

            chartControl.AnimationStartMode = ChartAnimationMode.OnLoad
        End Sub

        Private Sub buttonOnClick(ByVal sender As Object, ByVal e As EventArgs) Handles btnAnimate.Click
            chartControl.Animate()
        End Sub
        #End Region ' #AnimationConfig
    End Class
End Namespace

File: CS/AnimationExample/MainForm.cs
using DevExpress.XtraCharts;
using System;
using System.Windows.Forms;

namespace AnimationExample {
    public partial class MainForm : DevExpress.XtraEditors.XtraForm {
        Series series { get { return chartControl.Series[0]; } }
        PieSeriesView view { get { return series.View as PieSeriesView; } }

        public MainForm() {
            InitializeComponent();
        }

        #region #AnimationConfig
        protected override void OnLoad(EventArgs e) {
            base.OnLoad(e);

            view.Animation = new GripPieAnimation {
                BeginTime = new TimeSpan(0, 0, 0, 0),
                PointDelay = new TimeSpan(0, 0, 0, 0, 250),
                Duration = new TimeSpan(0, 0, 0, 1, 500),
                EasingFunction = new LinearEasingFunction {
                    EasingMode = EasingMode.Out
                }
            };

            chartControl.AnimationStartMode = ChartAnimationMode.OnLoad;
        }

        private void buttonOnClick(object sender, EventArgs e) {
            chartControl.Animate();
        }
        #endregion #AnimationConfig
    }
}

